src/conda/env_ENSO_RWS.yml

```diff
@@ -1,13 +1,18 @@
-name: _MDTF_ENSO_MSE
```

Comment:
You can delete this file--I will update the NCL and MDTF base yamls with nco and subprocess, respectively. Just add nco to the POD settings file in the ncl array in the runtime_requirements section.